### PR TITLE
fix reset password and table slice

### PR DESCRIPTION
**Pull request by merlouduron**
# Pull Request Review Comments

## Overview
- **Purpose of the PR**:
    - [ ] Feature addition
    - [ ] Bug fix
    - [ ] Refactoring
    - [ ] Documentation update
    - [ ] Other (please specify below)

## Specific Areas for Verification
- Please specify any particular requests or questions for the reviewer here.

## Suggestions/Concerns
- If you have any suggestions or concerns, please mention them here.

---

_**Additional Comments for the Reviewer (Optional)**_

- Add any additional comments or information for the reviewer here.

### CODE REVIEW

File: client/src/reducers/table/tableSlice.ts [hunk 1]
```diff
@@ -8,8 +8,8 @@ const initialState = {
 }
```
Comment (edward-stalley-work) [inline client/src/reducers/table/tableSlice.ts:8]: I get errors on dashboard and planning.
Do i need to rebuild DB or do something?

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
please see comment!

File: backend/api/views.py [hunk 2]
```diff
@@ -863,7 +863,7 @@ def post(self, request):
         # Send password reset email
         subject = _('Password Reset Request')
         message = _('Please click the link below to reset your password:\n\n') + reset_url
-        from_email = 'MS_zAbrXd@trial-o65qngkpyq3lwr12.mlsender.net'  # Update with your email
+        from_email = 'shininoue81@gmail.com'  # Update with your email
```
Comment (edward-stalley-work) [inline backend/api/views.py:866]: So should this be changed to admin email or Datt japan email in the future or what should we do with this exactly?
Or generic name as it is just for sending a link?

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
please see comment

File: client/src/reducers/table/tableSlice.ts (hunk 1 above)
Comment (edward-stalley-work) [reply client/src/reducers/table/tableSlice.ts:8]: fixed

File: backend/api/views.py (hunk 2 above)
Comment (merlouduron) [reply backend/api/views.py:866]: This must be change and use the official email to be used for reset password. The software I'm using is Mailjet. You can create a another gmail account , create Mailjet account using the newly created gmail account and use that gmail to setup on Mailjet. That way, you can change from_email.

Comment (edward-stalley-work) [reply backend/api/views.py:866]: okay gotcha - thank you for letting me know.

**Review by edward-stalley-work — APPROVED** [review]

**Pull request merged**